spec/text-editor-component-spec.js

```diff
@@ -3768,7 +3772,11 @@ describe('TextEditorComponent', () => {
       const { item: later, decoration } = createBlockDecorationAtScreenRow(
         editor,
         4,
-        { height: 20, position: 'after', order: 3 }
+        {
```

Comment:
All this file changes is formatting. Discarding this would make the Run Linter fail, right?

Reply:
https://prettier.io/docs/en/rationale.html#multi-line-objects This doesn't make sense

Reply:
Its saying it preserves the single/multi line layout to a reasonable extent. So you can revert the change and the lint should still pass. If in doubt, just run `script/lint`.

Reply:
Okay.

Reply:
By the way, in my experience, CI doesn't error out on any "badly formatted code" in files you didn't change in your PR.

(I don't know how this works exactly; Maybe some persistent cache of files that were already linted once and deemed okay? Or it's clever enough to look at only changed files from reading the git metadata?)